resources/js/Pages/Results.vue

```diff
@@ -194,6 +197,9 @@
             unexpectedError() {
                 const flashMessages = this.$page.props.flash as FlashMessages;
                 return (flashMessages.errors && flashMessages.errors.unexpected);
+            },
+            iconDirection() {
+                return document.querySelector('html')?.getAttribute('dir');
```

Comment:
So here we need to actually get the computed style rather that what the attribute was set to. It can be done with `window.getComputedStyle(document.documentRoot)`

Reply:
`window.getComputedStyle(document.body).dir` gives typescript errors.

Reply:
Cool that you could figure it out :)